.bazelrc

```diff
@@ -1,23 +1,14 @@
-common --noenable_bzlmod
-common --enable_workspace
-# Avoid being broken by version requirement changes in transitive deps.
-common --check_direct_dependencies=error
-
-common --enable_platform_specific_config
+# Import shared config
+try-import %workspace%/shared.bazelrc
```

Comment:
Should probably use a regular import here, not an optional import, since we expect the file to exist

Reply:
Hmm I was a bit iffy about using a stricter option. But I guess we do want to enforce it.

Changed.